Library/Homebrew/dev-cmd/edit.rb

```diff
@@ -70,9 +70,8 @@ def run
              !Homebrew::EnvConfig.no_env_hints? &&
              (core_formula_path?(path) || core_cask_path?(path) || core_formula_tap?(path) || core_cask_tap?(path))
            end
-          opoo <<~EOS
-            `brew install` ignores locally edited casks and formulae if
-            HOMEBREW_NO_INSTALL_FROM_API is not set.
+          ohai "To test your local edits, run:", <<~EOS
+            HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source --verbose --debug #{args.named.join(" ")}
```

Comment:
```suggestion
          puts <<~EOS
            To test your local edits, run:
              HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source --verbose --debug #{args.named.join(" ")}
```

Reply:
I was lazy to make a screen in the first place, but here it is.

**`ohai`**

![image](https://github.com/user-attachments/assets/c876f25c-fb4d-4a38-8ccb-77167bac0d57)

**`puts`**

![image](https://github.com/user-attachments/assets/8c34ba27-be35-4ece-b4a1-de8bcc0f7635)

Both seems fine.